resource/css/skosmos.css

```diff
@@ -376,6 +376,10 @@ body {
     font-weight: bold;
   }
 
+  #sidebar, #main-content {
+    background-color: var(--vocab-bg);
```

Comment:
This affects now all main-content elements.
 - Maybe we need to either further divide the css definition to #main-content.vocabPage, #main-content.conceptPage
 - Or rename the vocab-bg to be something more generic, since leaving it like this would mean changing the color variable would change the lolours of at least vocab-info and error pages

Reply:
I moved the background-color setting to `.main-content-section` class, which is more specific and allows for multiple sections within the main-content area. I think it solves the problem well enough for now.